src/sdk.js

```diff
@@ -1,57 +1,99 @@
 window.Smartcar = (function(window) {
   'use strict';
+  // accesss denied error class
+  class AccessDenied extends Error { // eslint-disable-line no-restricted-syntax
+    constructor(message) {
+      super(message);
+      this.name = 'AccessDenied';
+    }
+  }
 
   /**
    * Initializes Smartcar class
    *
    * @param {Object} options - the SDK configuration object
    * @param {String} options.clientId - the applications' client id
-   * @param {String} options.redirectUri - the registered redirect uri of the application
+   * @param {String} options.redirectUri - the registered redirect uri of the
```

Comment:
`redirectURI` here would be just their target origin if they use Smartcar hosting, not `cdn.smartcar.com/redirect?origin=<target-origin-here>`. Any other/preferred naming conventions?

Reply:
switching to redirectUri just being exactly what's on the dashboard as discussed in person